DOCS-3600 Update Rsync Task Article (#1345)
truenas/documentation: content/CORE/CORETutorials/Tasks/CreatingRsyncTasks.md

To create the connection, go to **System > SSH Connections** and click **ADD**.
Configure a **Semi-automatic** connection and set **Private Key** to *Generate New*.

{{< expand "Can this be set up in a command line instead?" "v" >}}
Go to the **Shell** on the **Host** system.
When a TrueNAS account other than *root* manages the rsync task, enter `su - USERNAME`, where *USERNAME* is the TrueNAS user account that runs the rsync task.
Enter `ssh-keygen -t rsa` to create the key pair.
When prompted for a password, press <kbd>Enter</kbd> without setting a password (a password breaks the automated task).
Here is an example of running the command:

```zsh
truenas# ssh-keygen -t rsa
Generating public/private rsa key pair.
Enter file in which to save the key (/root/.ssh/id_rsa):
Created directory '/root/.ssh'.
Enter passphrase (empty for no passphrase):
Enter the same passphrase again:
Your identification has been saved in /root/.ssh/id_rsa.
Your public key has been saved in /root/.ssh/id_rsa.pub.
The key fingerprint is:
SHA256:NZMgbuPvTHeEqi3SA/U5wW8un6AWrx8ZsRQdbJJHmR4 tester@truenas.local

> The key's randomart image is:

Reviewer: ```suggestion
The key randomart image is:
```

|     o oB+. .    |
|    . =Bo+.o     |
|     o+==oo      |
+----[SHA256]-----+
```
The default public key location is <file>\~/.ssh/id_rsa.pub</file>.
Enter `cat ~/.ssh/id_rsa.pub` to see the key and copy the file contents.
Copy it to the corresponding user account on the **Remote** system in **Accounts > Users**.
Click **EDIT** and paste the key into **SSH Public Key**.

Next, copy the host key from the **Remote** system to the **Host** system user's <file>.ssh/known_hosts</file> directory, using `ssh-keyscan`.

On the **Host** system, open the **Shell** and enter `ssh-keyscan -t rsa {remoteIPaddress} >> {userknown_hostsDir}` where *remoteIPaddress* is the **Remote** system IP address and *userknown_hostsDir* is the <file>known_hosts</file> directory on the **Host** system.
Example: `ssh-keyscan -t rsa 192.168.2.6 >> /root/.ssh/known_hosts`.
{{< /expand >}}

### SSH Mode Process

Go to **Tasks > Rsync Tasks** and click **ADD**.

![TasksRsyncTasksAddModeSSH](/images/CORE/12.0/TasksRsyncTasksAddModeSSH.png "Rsync Task: SSH Mode")

Configure the SSH settings first by selecting **SSH** in the **Rsync Mode** dropdown and entering the **Port** number and **Remote Path**.

> Next, define the **Source** dataset for the rsync task and select a *User* account.

Reviewer: In this and the next line 113 the formatting on user. in this sentence are you talking about the field **User** or the user in general? if the later then it is just user. if the former then make that clear 
```suggestion
Next, define the **Source** dataset for the rsync task and select an account in **User**.
```

The **User** must be identical to the [SSH Connection]({{< relref "CORE/CORETutorials/SystemConfiguration/ConfiguringSSHConnections" >}}) **Username**.

Choose a direction for the rsync task, either **Push** or **Pull**, and define the task **Schedule**.

> Next, enter the **Remote** host IP address or hostname.

Reviewer: host name unless talking about a command or part of a command string that uses hostname
```suggestion
Next, enter the **Remote** host IP address or host name.
```
